src/pg/sql/11_cdb_union_adjacent.sql

```diff
@@ -0,0 +1,43 @@
+CREATE OR REPLACE FUNCTION _cdb_final_union_adjacent( joined_geoms geometry[] )
+RETURNS geometry[] AS $$
+BEGIN
+    RETURN joined_geoms;
```

Comment:
We should `unnest` here so the user does not have to, which would also mean changing the `RETURNS` type to `geometry`.